src/app/interfaces/IProfesional.ts

```diff
@@ -79,6 +79,11 @@ export interface IProfesional {
         fechaEgreso: Date;
         fechaTitulo: Date;
         renovacion: boolean;
+        renovacionOnline: {
```

Comment:
Debería llevar '?' porque no es requerido y fecha es Date.
```suggestion
        renovacionOnline?: {
            estado: String;
            descripcion: String;
            fecha: Date;
        };
```

Reply:
corregido!